.github/workflows/update-readme.yml

```diff
@@ -21,5 +21,17 @@ jobs:
       - name: Compile parsers
         run: ./nvim.appimage --headless -c "TSInstallSync all" -c "q"
```

Comment:
Skip that? Or do we want to add information to the README/docs that requires parser information?

Reply:
I don't think we need that, you can remove it